sprocs/check_assessment_access.sql

```diff
@@ -20,10 +20,13 @@ CREATE OR REPLACE FUNCTION
         OUT seb_config JSONB,         -- SEBKeys (if any) for this assessment.
         OUT show_closed_assessment boolean, -- If students can view the assessment after it is closed.
         OUT show_closed_assessment_score boolean, -- If students can view their grade after the assessment is closed
+        OUT submittable boolean,     -- If the assessment is visible but not submittable
         OUT access_rules JSONB       -- For display to the user. The currently active rule is marked by 'active' = TRUE.
     ) AS $$
 DECLARE
     active_access_rule_id bigint;
+    assessment_start_date TIMESTAMP WITH TIME ZONE;
+    assessment_end_date TIMESTAMP WITH TIME ZONE;
```

Comment:
Not sure I understand why these are needed...

Reply:
Oops, this was left over from my previous work. It shouldn't be there.